modules/virt-delete-vm-web.adoc

```diff
@@ -1,22 +1,33 @@
 // Module included in the following assemblies:
 //
-// * virt/virtual_machines/virt-delete-vms.adoc
+// * virt/managing-vms/virt-delete-vms.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="virt-delete-vm-web_{context}"]
 
 = Deleting a virtual machine using the web console
 
-Deleting a virtual machine permanently removes it from the cluster.
+Deleting a virtual machine (VM) permanently removes it from the cluster.
+
+If the VM is delete protected, the *Delete* action is disabled in the VM's *Actions* menu. To delete the VM you must first disable the VM's delete protection setting.
+
+
 
```

Comment:
Is the thing mentioned above about clearing a VM's delete protection setting something that is done in this procedure, or is it done elsewhere? If elsewhere, should this be added as a prereq below?